src/pages/en/migrate.md

```diff
@@ -63,19 +63,21 @@ Read [RFC0017](https://github.com/withastro/rfcs/blob/main/proposals/0017-markdo
 
 This includes a few changes to be aware of:
 
-- **BREAKING:** `<script hoist>` is the new default `<script>` behavior. The `hoist` attribute has been removed.
+- **BREAKING:** `<script hoist type="module">` is the new default `<script>` behavior. The `hoist` and `type="module"` attributes have been removed.
```

Comment:
```suggestion
- **BREAKING:** `<script hoist>` and `<script type="module">` is the new default `<script>` behavior. Both `hoist` and `type="module"` attributes have been removed. 
You can still use `type="module"` however this would be treated as `<script is:inline>`
```